vulcanai2/datasets/utils.py

```diff
@@ -107,6 +112,15 @@ def __call__(self, data):
 
 #THIS IS FROM SNEHA  https://github.com/sneha-desai
 def stitch_datasets(df_list, on, index_list=None):
```

Comment:
This will need to be updated with what Joseph uses for dataset stitching.

Reply:
it's too slow and not able to handle larger datasets in it's current state

Reply:
I can't find that, so I'll just ask Joseph to add it in at the end.